docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
+$$C = d_{NIS}/(d_{NIS}+d_{PIS}).$$
+
+The quantity $C$ measures the closeness of a particular alternative to the worst and best alternatives, such that the higher $C$, the closer to the best alternative and further from the worst. This method allows alternatives which perform poorly under some criteria to balance their ranking with a high performance in other criteria, providing a measure of trade-offs between criteria.
+
+VIKOR is another very popular MCDA algorithm, which incorporates two distance formulations; the Manhattan distance (or $L^1$ distance) and Chebyshev distance. The Manhattan distance, $S$, for each alternative is the sum over all criteria of the distance between each criteria’s value and its $PIS$, giving a measure of the overall utility of each alternative when considering all criteria. 
+
+$$S_i = \sum_{j=1}^M \frac{PIS_j-X_{i,j}}{PIS_j-NIS_j}$$
+
+I.e. this is the general performance of an alternative when considering all criteria, with possible trade-offs. The Chebyshev distance, $R$, is the maximum distance between the PIS and the value of a criteria for each alternative, giving a measure of the maximum deviance from the best solution for that alternative. 
+
+$$R_i = \textbf{max}_{j} \frac{PIS_j-X_{i,j}}{PIS_j-NIS_j}$$
+
+I.e. this gives a measure of the worst hidden trade-off in the Manhattan Distance (which is not considered in TOPSIS). The final ranking $Q$ is calculated from a linear combination of $R$ and $S$ where the weighting of $R$ and $S$ are determined by the balance of ‘group utility’ (measured by $S$) and ‘individual regret’ (measured by $R$) a decision maker wants to incorporate in their decision solution.
+
+$$S^- = \textbf{min}_{i}(S_i),S^+ = \textbf{max}_{i}(S_i)$$
+$$R^- = \textbf{min}_{i}(R_i),R^+ = \textbf{max}_{i}(R_i)$$
+$$Q_i = \delta \frac{S_i-S^-}{S^+-S^-} + (1-\delta)\frac{R_i-R^-}{R^+-R^-}$$
+
+Here, $\delta$ is a weighting which balances strategy between maximising all criteria and regret due to the worst performing criteria. Additional steps to the algorithm give guidelines for choosing between a single best alternative and n (n>1) best alternatives.
+
+## Post-ranking distance sorting 
+If highly ranked sites happen to be very close together in a particular domain, this can represent an issue for site selection. This is because the site selection algorithm will seed or shade in a very small region, representing a significant loss of resources if a high damage, localised event, such as a ship grounding, were to occur. To represent an insurance policy against this possibility, the user can also specify distance thresholds for selected sites. If sites are selected which are too close, sites will be replaced by the next highest ranked sites which are further apart until the distance threshold is satisfied. The distance sorting parameter inputs are described in the table below:
+
+|Parameter                    |Description                                                                        |
```

Comment:
Introduce/mention these in the table above first, and then go into more detail here.